### ISSUE TITLE
Bad Query

### ISSUE THREAD

**Issue by Eugenio350** [sqlitebrowser/dbhub.io#318]
I ran a query from https://dbhub.io/eugeniofelipe.perez, which was faulty and very computationally expensive, after this i am unable to access the home page of my account and my data. Probably because the query is still running and will continue to run till the end of time. I request politely for you guys to stop the query manually on my behalf so the account remains usable and i can access my data.

The name of the database is output_database_all.db

**Comment by justinclift** [comment]
Ahhh, that's interesting.  Haven't had that happen before.

Looking at the backend node that was assigned to handle requests for your database, it's showing SQL logic errors that aborted early, and it's not showing anything still running.

Looks like the database was successfully deleted too.  Did it come good at some point and let you do things with it again?

**Comment by justinclift** [comment]
Am trying to think how we could potentially catch the problem early, so things are more resilient.  Will need to think on it and probably try breaking it in a similar way. :smile:

**Comment by Eugenio350** [comment]
Yeah after around 8 hours the query ended (During this timeframe i had no access to the db), i thought it was you guys, well all resolved. To break it i tried grouping all rows by all different columns (72) on 3 million observations, that should do the trick. Sidenote: i needed to identify duplicates and there were no unique identificators.

**Comment by justinclift** [comment]
Interesting.  We'll probably need to trigger the same behaviour in a test environment and see why it's freezing things out for you.  In theory it shouldn't, but clearly the theory needs work. :wink:
